pkg/utils/utils.go

```diff
@@ -110,8 +117,8 @@ func DeleteSyncSet(name string, namespace string, client client.Client, reqLogge
 	}
 
 	// Only delete the syncset, this is just cleanup of the synced secret.
-	// The ClusterDeployment controller manages deletion of the pagerduty serivce.
-	reqLogger.Info("Deleting SyncSet", "Namespace", namespace, "Name", name)
+	// The ClusterDeployment controller manages deletion of the deadmanssnitch serivce.
+	log.Info("Deleting SyncSet", "Namespace", namespace, "Name", name, "DeadMansSnitchIntegreation.Namespace", dmsi.Namespace, "DMSI.Name", dmsi.Name, "cluster-deployment.Name:", clustDeploy.Name, "cluster-deployment.Namespace:", clustDeploy.Namespace)
```

Comment:
I think these additional name/namespace values are empty here, since you just created empty structs on L103 and L104. Am I right?

Reply:
Oops sorry im adding more to it now

Reply:
This is actually not needed as there are logs for syncsets already in the controller. That tell you the dmsi and cluster deployment